bundle-size/test/api.test.js

```diff
@@ -33,25 +33,29 @@ describe('bundle-size api', () => {
   let probot;
   let app;
   const db = dbConnect();
-  const nodeCache = new NodeCache();
+  let nodeCache;
+  let logWarnSpy;
 
-  beforeAll(async () => {
+  beforeAll(async done => {
```

Comment:
Why was this change needed?

Reply:
Made these tweaks when I was trying to find the culprit preventing Jest from exiting properly. Reverting (the node cache was the issue)